### ISSUE TITLE
Get-NetFirewallApplicationFilter: Access is denied.

### ISSUE THREAD

**Issue by ciumete** [RainbowMiner/RainbowMiner#2739]
UAC is disabled.
start.bat starting with windows (added start.bat link in shell:startup)
When mining process starting the console writes Get-NetFirewallApplicationFilter: Access is denied. and creating firewall rules in infinite loop...

**Comment by whitindllc** [comment]
right click and run the process as administrator?

**Comment by ciumete** [comment]
> right click and run the process as administrator?

if i set to startup with administrator privileges, when i restart the windows, don't run startup link...
but the user is on administrator group !!!

**Comment by whitindllc** [comment]
so you have a scheduled task to run start.bat but it doesnt run

you need to run it with the user SYSTEM

**Comment by ciumete** [comment]
> so you have a scheduled task to run start.bat but it doesnt run
> 
> you need to run it with the user SYSTEM

No, is not a scheduled task. I have added the start.bat link in startup section in windows (run -> shell:startup)

**Comment by whitindllc** [comment]
i have it running as a scheduled task and it works fine... i dont trust startup task

**Comment by ciumete** [comment]
> i have it running as a scheduled task and it works fine... i dont trust startup task

so nobody have a solution to run start.bat with windows startup task ...

**Comment by whitindllc** [comment]
go to google and look at schtasks command and switches.... 
essentially ditch WTF you are trying to do with msconfig and windows startup... it wont work unless you create a service and start it as system... the work around is to create a scheduled task...
open command prompt as admin and copy the below command.... it might need tweaking but these are most of the switches.. obviously replace c:\WhereEverFolderRBMis\ with whatever the path to your RBM folder is and then \start.bat

schtasks /create /tn "Mine Mfer" /tr c:\WhereEverFolderRBMis\start.bat /sc onstart /sd 04/08/2024 /ed 01/05/2050 /ru system

**Comment by ciumete** [comment]
> go to google and look at schtasks command and switches.... essentially ditch WTF you are trying to do with msconfig and windows startup... it wont work unless you create a service and start it as system... the work around is to create a scheduled task... open command prompt as admin and copy the below command.... it might need tweaking but these are most of the switches.. obviously replace c:\WhereEverFolderRBMis\ with whatever the path to your RBM folder is and then \start.bat
> 
> schtasks /create /tn "Mine Mfer" /tr c:\WhereEverFolderRBMis\start.bat /sc onstart /sd 04/08/2024 /ed 01/05/2050 /ru system

i try to do same what do task scheduler but with startup...

**Comment by whitindllc** [comment]
are you creating it with Admin privileges?

**Comment by ciumete** [comment]
> are you creating it with Admin privileges?

if i create with admin privileges don't start when the windows start...

**Comment by ciumete** [comment]
Username is in Administrators group , have no password.

I tryed with schedule tasks:
If i using SYSTEM account start.bat run but all processes are in background, can't see anything!
If i using Username account and start.bat is marked "Run as administrator" the task don't start at startup or logon
If i using Username account and start.bat is not marked "Run as administrator" the task run but i receive message "Get-NetFirewallApplicationFilter: Access is denied."

So is same behavoir like windows startup ...

**Comment by whitindllc** [comment]
oh thats in the config..... find this line and see whats there

 "ShowMinerWindow": "1",

**Comment by ciumete** [comment]
> oh thats in the config..... find this line and see whats there
> 
> "ShowMinerWindow": "1",

The config is ok, is "ShowMinerWindow": "1",
If i start a process in system account and i am logged with another account seems logic to not see that process !
